lib/runtime/addon.js

```diff
@@ -341,4 +340,34 @@ export default class Addon {
     return configModule && (configModule.default || configModule);
   }
 
+  get blueprintNames() {
+    let names = fs.readdirSync(path.join(this.dir, 'blueprints'));
+    return names;
```

Comment:
We might need to filter this for directories only, otherwise you might end up with blueprints like `.DS_Store`